### PR TITLE
Use webpack 5 and friends

### PR DESCRIPTION
**Pull request by StykMartin**
We have some technical debt here. Let's remove it.
Moving to latest greatest versions.

Signed-off-by: Martin Styk <mart.styk@gmail.com>

### CODE REVIEW

File: src/store/ducks/rootSaga.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ import { all, takeLatest } from 'redux-saga/effects';
 import load from './localtime/sagas';
 import { LocaltimeTypes } from './localtime/types';
 
-export default function* rootSaga() {
+export default function* rootSaga(): any {
```
Comment (renanrodrigo) [inline src/store/ducks/rootSaga.ts:6]: Can those (this and `load*`) be typed as `Generator` instead of `any`?

Comment (StykMartin) [reply src/store/ducks/rootSaga.ts:6]: We will end up w/ any somewhere, I can put `Generator` here and `load()`, however, I will need to explicitly cast `const response = yield call()` to `any`

Comment (renanrodrigo) [reply src/store/ducks/rootSaga.ts:6]: I hate it. Let it be `any` here then, no need to get trouble for nothing

**Review by renanrodrigo — APPROVED** [review]
big upgrade so we can break many things at once - thanks!

**Comment by StykMartin** [comment]
Fail fast, fail often 🐬

**Pull request merged**